mongodbatlas/data_source_mongodbatlas_online_archive.go

```diff
@@ -83,6 +83,10 @@ func schemaOnlineArchive() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Computed: true,
 		},
+		"collection_type": {
+			Type:     schema.TypeString,
+			Computed: true,
```

Comment:
also looks like we are missing _test.go file, i.e. data_source_mongodbatlas_online_archive_test.go. if true can you add it as part of this PR?

Reply:
same resource has data source tests combined will fix in later release